### PR TITLE
Feature: Proposal creation

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description

This is the first pass of proposal creation. There are a few caveats though as listed below:
- ~~Actions can't be added yet(we need helper methods from the SDK to do this)~~ Withdraw action is already available 
- Trying to create a proposal in WhitelistVoting DAOs will fail without any UI feedback if your wallet is not in the whitelist
- Metadata isn't handled(needs the IPFS client)
- VotingTerminal data and Setup Voting data are hardcoded. We need to read that from subgraph 

Task: [APP-81](https://aragonassociation.atlassian.net/jira/software/c/projects/APP/boards/40?modal=detail&selectedIssue=APP-81)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/pages/newProposal.tsx [hunk 1]
```diff
@@ -37,14 +42,111 @@ const NewProposal: React.FC = () => {
...
+  //   estimateCreationFees,
+  //   shouldPoll
+  // );
+
+  const handleCloseModal = () => {
+    switch (creationProcessState) {
+      case TransactionState.LOADING:
+        break;
+      case TransactionState.SUCCESS:
+        navigate(Landing);
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/newProposal.tsx]: This didn't navigate for me. Also, I think the navigation should be to the Governance page, no?

File: packages/web-app/src/containers/transactionModals/publishModal.tsx [hunk 2]
```diff
@@ -31,7 +33,7 @@ const icons = {
   [TransactionState.ERROR]: <IconReload />,
 };
 
-const PublishDaoModal: React.FC<PublishDaoModalProps> = ({
+const PublishModal: React.FC<PublishModalProps> = ({
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/transactionModals/publishModal.tsx:37]: Thanks for extending and refactoring this modal 👍🏾

**Review by Fabricevladimir — APPROVED** [review]
Thanks, can't wait until we get the ipfs stuff to work

**Comment by Fabricevladimir** [comment]
Quick question. I saw that the withdraw action is already implemented on the SDK, is it not yet configured within a proposal?

**Comment by RakeshUP** [comment]
@Fabricevladimir Just saw the withdraw action. Will try to add to this PR or do it in a separate PR as there are some more tasks to be done like querying subgraph for voting data in step 2(quorom, minParticipation, etc)

**Review by WissenIstNacht — APPROVED** [review]
Nice, thanks!

**Pull request merged**